### ISSUE TITLE
If you have an elm-package with BSD3 as the license, it fails

### ISSUE THREAD

**Issue by eeue56** [avh4/elm-upgrade#30]
Setups:

- Grab https://github.com/eeue56/elm-lazy master
- Run elm-upgrade
- It will fail due to the license being BSD3, not BSD-3-Clause

Would be cool if it automatically converted that to BSD-3-Clause (or fail early as in #28 )

**Comment by avh4** [comment]
Fixed in b780fc5, will be included in `0.19.0-rc7`

**Comment by avh4** [comment]
Info message is:
```
INFO: Detected 'BSD3' license, which is not a valid SPDX license identifier; converting to 'BSD-3-Clause'
```
